anti-analysis/anti-forensic/clear-logs/clear-the-windows-event-log.yml

```diff
@@ -3,13 +3,17 @@ rule:
     name: clear the Windows event log
     namespace: anti-analysis/anti-forensic/clear-logs
     author: michael.hunhoff@fireeye.com
-    scope: basic block
+    scope: function
     att&ck:
       - Defense Evasion::Indicator Removal on Host::Clear Windows Event Logs [T1070.001]
     examples:
       - 82BF6347ACF15E5D883715DC289D8A2B:0x14005E0C0
+      - mimikatz.exe_:0x45228B
   features:
     - and:
-      - api: advapi32.ElfClearEventLogFile
+      - or:
+        - api: advapi32.ElfClearEventLogFile
+        - api: advapi32.ClearEventLog
       - optional:
         - api: advapi32.OpenEventLog
+        - api: advapi32.GetNumberOfEventLogRecords
```

Comment:
yeah, i like this. its generally related to event log manipulation, and especially until this lives in its own rule, is the best place to describe the behavior. thanks for including it here.